### DISCUSSION TITLE
General

### DISCUSSION THREAD

**Discussion by TheCaduceus** [TheCaduceus/tg-upload#15] [General]
Chat about anything and everything here.

**Comment by joliejoker** [comment]
Hi Mr., I'm sorry to bother you, but I love your tg-upload script...
and your others projects.
Could you explain something more about it?
I want learn more about python!

**Comment by 0xWeverton** [comment]
Firstly, I thank you for the project, but I have a question. When using the function to upload files, is there any way to keep the files in a queue and send them all at once? I know that the limit per time is 10, but is there any way to send 10 videos, for example, together? I noticed that using the flags mentioned in the readme, it only sends one at a time. To be clearer, it's something like this: (the name of the option in Telegram is 'group items').

![image](https://github.com/TheCaduceus/tg-upload/assets/57471470/fd73d0c3-eb36-4b79-93f9-0f58c6685946)

**Reply by TheCaduceus** [reply]
tg-upload does not support sending files as album yet.
